airbyte-integrations/connectors/source-file/source_file/client.py

```diff
@@ -96,7 +96,7 @@ def close(self):
 
     def backoff_giveup(self, error):
         # https://github.com/airbytehq/oncall/issues/1954
-        if isinstance(error, SSHException) and str(error) == "Error reading SSH protocol banner":
+        if isinstance(error, SSHException) and str(error).startswith("Error reading SSH protocol banner"):
```

Comment:
Should we have a test changing in that case?

Reply:
improved